Python/pywarpx/picmi.py

```diff
@@ -355,6 +355,13 @@ def init(self, kw):
         self.max_grid_size = kw.pop('warpx_max_grid_size', 32)
         self.blocking_factor = kw.pop('warpx_blocking_factor', None)
 
+        self.potential_xmin = 0
+        self.potential_xmax = 0
+        self.potential_ymin = kw.pop('warpx_potential_zmin', 0)
+        self.potential_ymax = kw.pop('warpx_potential_zmax', 0)
```

Comment:
Is it standard practice in WarpX that input parameters be named in terms of X/Y/Z regardless of using RZ geometry? I understand why the internal variables are named as they are, to avoid conditionals elsewhere.